### PR TITLE
DOC-4240: Documentation for Mark all as read in All Notifications page

### PR DESCRIPTION
**Pull request by mzorai**

### CODE REVIEW

**Review by hghorbel — APPROVED** [review]

File: docs/PersonalApplications.rst [hunk 1]
```diff
@@ -787,6 +787,15 @@ notifications. Besides, the **Mark all as read** function allows you to
 change all messages on this menu into the read status. Finally, to
 remove any notification item, simply click the corresponding |image51|.
 
+.. note:: You can also view your all On-site notifications in the page ``My Notifications`` accessible by clicking on
```
Comment (plamarque) [inline docs/PersonalApplications.rst:790]: you can view all your notifications

File: docs/PersonalApplications.rst [hunk 2]
```diff
@@ -787,6 +787,15 @@ notifications. Besides, the **Mark all as read** function allows you to
...
+          your username the on My Notifications:
```
Comment (plamarque) [inline docs/PersonalApplications.rst:791]: then on My Notifications

File: docs/PersonalApplications.rst [hunk 3]
```diff
@@ -787,6 +787,15 @@ notifications. Besides, the **Mark all as read** function allows you to
...
+
+            |image90|
+            
+          It is possible to mark your unread notifications as read simply by clinking on ``Mark all as read`` in the top of the page:
```
Comment (plamarque) [inline docs/PersonalApplications.rst:795]: to mark all your notifications as read simply by clicking

**Review by plamarque — COMMENTED** [review]
Several typos

**Pull request merged**